test/libsolidity/util/BytesUtils.cpp

```diff
@@ -206,10 +217,33 @@ string BytesUtils::formatString(bytes const& _bytes, size_t _cutOff)
 	return os.str();
 }
 
+std::string BytesUtils::formatFixedPoint(bytes const& _bytes, bool _signed, size_t _fractionalDigits)
+{
+	string value;
+	bool negative = false;
+	if (_signed)
+	{
+		s256 signedValue{u2s(u256{_bytes})};
```

Comment:
I think these here should also all use `fromBigEndian`

Reply:
True